src/main/features/response/views/statement-of-means/residence.njk

```diff
@@ -0,0 +1,55 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, radioOption, saveAndContinueButton, textInput %}
+
+{% set title = 'Money claim' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('Where are you living?') }}</h1>
+
+      <form
+        method="POST"
+        class="analytics-submit-event-trigger"
+        data-event-action="Response: Statement of means: Select defendant response option"
```

Comment:
An action does not sound good here. We asking for `Where are you living?` while we sending an action to GA as `Select defendant response option`